cpp/src/quantiles/tdigest/tdigest_aggregation.cu

```diff
@@ -240,11 +279,23 @@ struct cumulative_centroid_weight {
   GroupOffsetsIter group_offsets;    // groups
   cudf::device_span<size_type const> tdigest_offsets;  // tdigests with a group
 
+  cumulative_centroid_weight(double const* _cumulative_weights,
```

Comment:
I don't think we need this constructor as we can just use brace initializations or even better designated initializers (now that we have C++20) which the constructor would actually prevent